File: d356(series)/Python/series.py
from sys import stdin, stdout

while True:
    k = stdin.readline()
    if not k: break
    k = int(k)
    count, sum = 0, 0

    while k >= sum:
        count += 1
        sum += 1 / count
    stdout.write(str(count) + '\n')

File: d356(series)/C++/series.cpp
#include <iostream>

int main(void) {
  int k;
  while (std::cin >> k) {
    double sum = 0, count = 0;
    while (k >= sum) {
      count++;
      sum += 1 / count;
    }
    std::cout << count << '\n';
  }
  return 0;
}
